packages/analytics/analytics-chart/src/components/SimpleChart.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="analytics-chart-shell">
+    <KEmptyState
+      v-if="!hasValidChartData"
+      class="chart-empty-state"
+      :cta-is-hidden="true"
+      data-testid="no-data-in-report"
+      icon="stateNoData"
+      icon-size="170"
+    >
+      <template #title>
+        {{ emptyStateTitle }}
+      </template>
+      <template #message>
+        {{ emptyStateDescription }}
+      </template>
+    </KEmptyState>
+    <div
+      v-else
+      class="analytics-chart-parent"
+    >
+      <DoughnutChart
+        v-if="isGaugeChart"
+        :chart-data="computedChartData"
+        :dataset-colors="chartOptions.chartDatasetColors"
+        is-simple
+        :legend-position="legendPosition"
+        :legend-values="legendValues"
+        :metric-display="chartOptions.metricDisplay"
+        :metric-unit="computedMetricUnit"
+        :synthetics-data-key="syntheticsDataKey"
+        :width="width"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import composables from '../composables'
+import { AnalyticsChartOptions } from '../types'
+import { ChartTypesSimple, ChartLegendPosition } from '../enums'
+import DoughnutChart from './chart-types/DoughnutChart.vue'
+import { computed, PropType, provide, toRef } from 'vue'
+import { AnalyticsExploreResult, AnalyticsExploreV2Result } from '@kong-ui-public/analytics-utilities'
+import { datavisPalette } from '../utils'
+
+const props = defineProps({
+  chartData: {
+    type: Object as PropType<AnalyticsExploreResult | AnalyticsExploreV2Result>,
+    required: true,
+  },
+  chartOptions: {
+    type: Object as PropType<AnalyticsChartOptions>,
+    required: true,
+  },
+  emptyStateTitle: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  emptyStateDescription: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  legendPosition: {
+    type: String as PropType<`${ChartLegendPosition}`>,
+    required: false,
+    default: ChartLegendPosition.Right,
```

Comment:
Do the simple chart types have a concept of legend position?

Reply:
Unfortunately, yes. 

![mesh](https://github.com/Kong/public-ui-components/assets/103061463/07381f21-ea98-4366-bc91-fff2df3236ad)

Reply:
We decided to push off the legend display / positioning thing to a future PR, since it does not apply to GaugeChart.

the "Mesh Manager look" is just a shrunken DoughnutChart (since it's a "high fidelity" chart, with 2+ value datasets, legend, and custom title position). We just need to iterate w/ UX team on how to unify the two designs.

cc @sichvoge @ryanmoore